go/prepare_workspace.inc

```diff
@@ -13,10 +13,17 @@
 # limitations under the License.
 
 prepare_workspace() {
+	# Use Go modules (https://golang.org/ref/mod) if defined. This is the
+	# prefered workspace setup; it has been available since Go 1.11 and is the
+	# default behavior as of 1.16.
```

Comment:
indentation seems off. It looks like here it was done with `tab` but in the rest of the file the indentation was done with `spaces`

Reply:
thanks, fixed